### PR TITLE
Source code updates

### PR DESCRIPTION
**Pull request by tugraskan**
Tried to add Franks changes back, they were not carried over from Jeffs code.

### CODE REVIEW

File: src/soil_nutcarb_write.f90 [hunk 1]
```diff
@@ -65,33 +69,39 @@ subroutine soil_nutcarb_write(out_freq)
...
           soil_prof_hs = soil_prof_hs + soil1(j)%hs(ly)
           soil_prof_hp = soil_prof_hp + soil1(j)%hp(ly)
           soil_prof_microb = soil_prof_microb + soil1(j)%microb(ly)
           soil_prof_water = soil_prof_water + soil1(j)%water(ly)
         end do
-        ! This a test of push
-        ! soil1(j)%tot_org = soil_prof_hact + soil_prof_hsta + soil_prof_microb
-        soil1(j)%tot_org = soil_prof_hs + soil_prof_hp + soil_prof_microb + soil_prof_meta + &
-                           soil_prof_str + soil_prof_lig
+        soil1(j)%tot_org = soil_prof_hs + soil_prof_hp + soil_prof_microb
```
Comment (tugraskan) [inline src/soil_nutcarb_write.f90:89]: add meta too apparently

File: src/mgt_killop.f90
Comment (tugraskan) [inline src/mgt_killop.f90:1]: possible double counting

**Comment by fgeter** [comment]
Reviewed pull request and looks good.

**Comment by odav** [comment]
Successful merge.

**Pull request merged**